src/renderer/features/multisig-wallet-create/multisig-wallet/model/chain-model.ts

```diff
@@ -84,16 +81,27 @@ sample({
   target: $chainsWithFee,
 });
 
+const $chainsWithFeeAvailableForInitiatorChain = combine(
+  { chains: $chainsWithFee, initiator: flowModel.$initiator },
+  ({ chains, initiator }) => {
+    if (!initiator) return [];
+
+    return chains.filter(chain => accountService.isAccountAvailableOnChain(initiator, chain.chain));
+  },
+);
+
+const filteredChains = $chainsWithFeeAvailableForInitiatorChain.map(chains => chains.map(chain => chain.chain));
```

Comment:
first naming is very detailed and the second is very not 😅

```suggestion
const $multisigChains = combine(
  { chains: $chainsWithFee, initiator: flowModel.$initiator },
  ({ chains, initiator }) => {
    if (!initiator) return [];

    return chains.filter(chain => accountService.isAccountAvailableOnChain(initiator, chain.chain).map(chain => chain.chain));
  },
);
```